tests/acceptance/task-comments-test.js

```diff
@@ -15,14 +25,15 @@ test('Task comments are displayed correctly', function(assert) {
 
   server.createList('comment', 4, { taskId: task.id });
 
-  taskPage.visit({
+  renderPage();
+  page.visit({
```

Comment:
Since this is an acceptance test, you still want to test the `taskPage` by doing `.visit()`. Your `renderPage` is superfluous here because it's a) not an integration test, and b) we're testing the comments in the context of the task page (which is the parent, see below).

Reply:
@daveconnis 

As I spoke about during our call, integration tests, instead of booting up the whole app just do a `page.render`, to render the component they're testing.

Acceptance tests, however, are end to end and do boot-up the whole app. This is done in the `moduleForAcceptance` function.

That means that, instead of rendering that one component, since the app is booted up, you navigate to the page/route in the app which you are testing, using `page.visit`